go.mod

```diff
@@ -3,20 +3,20 @@ module github.com/crossplane/templating-controller
 go 1.13
 
 require (
-	github.com/crossplane/crossplane v0.8.0-rc.0.20200219030944-557074689d71
-	github.com/crossplane/crossplane-runtime v0.5.1-0.20200219022054-8948e50564b1
+	github.com/crossplane/crossplane v0.10.0-rc.0.20200429060016-7ced24cf6bfa
+	github.com/crossplane/crossplane-runtime v0.7.1-0.20200429054236-05ca13f8ea7b
 	github.com/google/go-cmp v0.4.0
 	github.com/pkg/errors v0.9.1
-	golang.org/x/net v0.0.0-20191028085509-fe3aa8a45271
+	golang.org/x/net v0.0.0-20200202094626-16171245cfb2
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
-	helm.sh/helm/v3 v3.0.0-20200205083830-5ec70ab27fbf
-	k8s.io/api v0.17.2
-	k8s.io/apiextensions-apiserver v0.17.2
-	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.2
-	sigs.k8s.io/controller-runtime v0.4.0
+	helm.sh/helm/v3 v3.2.0
```

Comment:
Do we have to update Helm?

Reply:
Build was failing before I updated, it is possible that there could be a workaround

Reply:
This was the error:
```
WARN [runner] Can't run linter unused: buildssa: analysis skipped: errors in package: [/home/dan/code/github.com/crossplaneio/templating-controller/pkg/operations/helm3/helm3.go:25:2: could not import hel
m.sh/helm/v3/pkg/action (/home/dan/code/github.com/crossplaneio/templating-controller/.work/pkg/pkg/mod/helm.sh/helm/v3@v3.0.0-20200205083830-5ec70ab27fbf/pkg/action/action.go:25:2: could not import k8s.i
o/cli-runtime/pkg/genericclioptions (/home/dan/code/github.com/crossplaneio/templating-controller/.work/pkg/pkg/mod/k8s.io/cli-runtime@v0.17.2/pkg/genericclioptions/builder_flags.go:23:2: could not import
 k8s.io/cli-runtime/pkg/resource (/home/dan/code/github.com/crossplaneio/templating-controller/.work/pkg/pkg/mod/k8s.io/cli-runtime@v0.17.2/pkg/resource/visitor.go:103:78: too few arguments in call to i.C
lient.Get().AbsPath("api", "v1", "namespaces", i.Namespace).Do)))] 
```

Reply:
Got it, probably k8s library breaking change.